src/bzflag/DisplayMenu.cxx

```diff
@@ -323,9 +331,14 @@ void            DisplayMenu::resize(int _width, int _height)
     if (renderer)
     {
         TextureManager& tm = TextureManager::instance();
-        ((HUDuiList*)listHUD[i++])->setIndex(BZDB.isTrue("dither"));
-        ((HUDuiList*)listHUD[i++])->setIndex(BZDB.isTrue("blend"));
-        ((HUDuiList*)listHUD[i++])->setIndex(BZDB.isTrue("smooth"));
+
+        // Quality
+        ((HUDuiList*)listHUD[i++])->setIndex(renderer->useQuality());
```

Comment:
OK, I see that this is an incredibly brittle coupling between the order of creation in the ctor and here. At the very least, this coupling should be documented, as in "hey make sure the order of access here matches the order of creation"

Probably outside the scope of what you were trying to accomplish, but I'm wondering why these mutation (`setIndex`) operations are happening here in `resize()`. It would be cleaner if the index values were set when the items are constructed.

Reply:
Yeah, that's outside the scope of this PR.  Our menu system as a whole needs a revamp or a replacement.